app/testOnly/controllers/StartJourneyController.scala

```diff
@@ -114,14 +127,27 @@ class StartJourneyController @Inject() (
       )).map(sjResponse => Redirect(sjResponse.nextUrl.value))
     }
   }
+  /**
+   * Pretends for testing purposes that journey started from EPAYE service
+   */
+  val startJourneyEpayeEpaye: Action[AnyContent] = as.default.async { implicit request =>
+    if (hc.sessionId.isEmpty) {
+      Future.successful(Ok("Missing session id"))
+    } else {
+      journeyConnector.Epaye.startJourneyEpayeService(SjRequest.Epaye.Simple(
+        returnUrl = ReturnUrl(routes.StartJourneyController.showBtaPage.url + "?return-page"),
+        backUrl   = BackUrl(routes.StartJourneyController.showBtaPage.url + "?starting-page")
+      )).map(sjResponse => Redirect(sjResponse.nextUrl.value))
+    }
+  }
 
   val startJourneyVatBta: Action[AnyContent] = as.default.async { implicit request =>
     if (hc.sessionId.isEmpty) {
       Future.successful(Ok("Missing session id"))
     } else {
       journeyConnector.Vat.startJourneyBta(SjRequest.Vat.Simple(
-        returnUrl = ReturnUrl(routes.StartJourneyController.showBtaPage.url + "?return-page"),
-        backUrl   = BackUrl(routes.StartJourneyController.showBtaPage.url + "?starting-page")
+        returnUrl = ReturnUrl(routes.StartJourneyController.showEpayePage.url + "?return-page"),
+        backUrl   = BackUrl(routes.StartJourneyController.showEpayePage.url + "?starting-page")
```

Comment:
this and the url above should have stayed the same shouldn't they?

Reply:
whoops